weasyl/moderation.py

```diff
@@ -718,3 +718,80 @@ def note_about(userid, target_user, title, message=None):
             userid=userid, target_user=target_user, unixtime=arrow.utcnow(),
             settings='s', content=staff_note,
         ))
+
+
+def audit_log(username=None, staff=None, start_date=None, end_date=None, timezone=None):
+    """Selects all the data needed to display the staff action audit log.
+    All arguments are optional and used to refine data.
+    Without a start and end date it is limited to 100 items.
+
+    Args:
+        username (str): Login name to filter by actions done to. Defaults to None.
+        staff (str): Login name to filter by actions done by. Defaults to None.
+        start_date (str): Show events after this date.
+            Accepts any string that arrow.get can parse. Defaults to None.
+        end_date (str): Show events before this date.
+            Accepts any string that arrow.get can parse. Defaults to None.
+        timezone (str): Timezone used to parse provided dates.
+
+    Returns:
+        A list of dicts with event data. Content is Markdown formatted.
```

Comment:
This is easy to misinterpret. Maybe “Content is rendered to HTML from Markdown” or something?

(Also, while you’re at it, please remove the blank lines after both docstrings.)